packages/aws-cdk-lib/aws-cloudfront-origins/lib/s3-bucket-origin.ts

```diff
@@ -105,20 +106,21 @@ export abstract class S3BucketOrigin extends cloudfront.OriginBase {
         };
       }
 
-      getBucketPolicyActions(accessLevels: cloudfront.AccessLevel[]) {
+      getBucketPolicyActions(accessLevels: Set<cloudfront.AccessLevel>) {
```

Comment:
This method and `getKeyPolicyActions` should be private I believe?

Reply:
Yes, thanks!